### DISCUSSION TITLE
Succession planning

### DISCUSSION THREAD

**Discussion by hildjj** [peggyjs/peggy#33] [General]
While I don't want this to turn into a money-making scheme (that doesn't feel right to do when someone else wrote the bulk of the code), I wouldn't mind a few small donations to defray DNS costs, etc. If we were to do this, I'd want to first discuss how to keep the money separate, what to do about taxes, and where any excess would go (e.g. a charity like https://girlswhocode.com/)

**Comment by phpnode** [comment]
Until this is set up I'm happy to cover the cost of DNS/Domain Name through my company - [codemix](https://codemix.com/). I'm not sure how we avoid having a single entity controlling the domain name though (I think we'd all like to avoid a repeat of the situation that led us here!)

**Reply by hildjj** [reply]
I registered for 3 years.  I think it helps that the domain points to github instead of a separate host; it's one less thing to ensure we've got redundancy on.

Yes, we need to figure out what happens if I get hit by a bus.  On the GitHub side, @linuxwolf is my designated successor; he'll have permissions and do the right thing if i literally die.  If I just stop responding, we'll keep a few extra people as admins so they can do the necessary.

For DNS, I've just made sure my wife has access to my dreamhost password, and I'll make sure she has instructions to hand off as needed (I'll make an intro over email).  That just leaves what happens if I stop responding.  What I *should* have done was start a new [insert DNS registrar] account, and shared the password with you.  Maybe we should do that when the name comes up for renewal?
